es/README.md

```diff
@@ -34,9 +34,9 @@ Bien, [empecemos por el comienzo...](./how_the_internet_works/README.md)
 
 ## Seguir el tutorial desde casa
 
-Participar en un taller de Django Girls en vivo es genial, pero somos conscientes de que no siempre es posible hacerlo. Por eso, te recomendamos hacer este tutorial en casa. Para las que estáis en casa, estamos preparando vídeos que facilitarán seguir el tutorial por tu cuenta. Todavía está en progreso, pero cada vez hay más cosas explicadas en el canal de YouTube [Coding is for girls](https://www.youtube.com/channel/UC0hNd2uW8jTR5K3KBzRuG2A/feed) (Nota: en inglés).
+Participar en un taller de Django Girls en vivo es genial, pero somos conscientes de que no siempre es posible hacerlo. Por eso, te recomendamos hacer este tutorial en casa. Para las que estáis en casa, estamos preparando videos que facilitarán seguir el tutorial por tu cuenta. Todavía está en progreso, pero cada vez hay más cosas explicadas en el canal de YouTube [Coding is for girls](https://www.youtube.com/channel/UC0hNd2uW8jTR5K3KBzRuG2A/feed) (Nota: en inglés).
```

Comment:
This translation is using Spanish from Spain:

`Para las que estáis en casa, estamos preparando...`

Vs what would be more normal in Latin America:

`Para las que están en casa, estamos preparando...`
`Para las que se encuentran en casa, estamos preparando...`

Both `video` and `vídeo` are correct uses of the word (that comes from english...) but in Spain the use the former, so in order to keep this in the same overall tone of Spanish (from Spain) I do not agree with this change.